.github/CONTRIBUTING.MD

```diff
@@ -0,0 +1,127 @@
+# Contributing to GSRI repositories
+
+This documentation provides guidelines and standard procedures to contribute to our project. It is a generic documentation which may be overriden on a per-project basis. Please check the `.guithub\CONTRIBUTING.md` file of projects for additional instructions.
+
+## Ways to contribute to GSRI projects
+
+There are two ways of contributing to our projects :
+
+1. Suggest ideas and feedback
+1. Contribute to the source code
+
+**Note:** in order to contribute to the source code you need to own a github.com account.
+
+## How to suggest ideas and feedback
+
+*We are in the process of writing this documentation*
```

Comment:
Issue and/or Discord message ?

Reply:
We need to take a decision on the standard way of handling feedback

Reply:
Action tracked by https://trello.com/c/T7dpBGoS

Reply:
Closing this one, will update the file on another PR when decision taken